api/helpers/game-states/create-socket-event.js

```diff
@@ -1,9 +1,9 @@
 const MoveType = require('../../../utils/MoveType.json');
 
 module.exports = {
-  friendlyName: 'Publish Game State',
+  friendlyName: 'Create Socket Event',
 
-  description: 'Determines if the game has ended and sends a socket event based on the game status.',
+  description: 'Creates a Socket event, and blast game over if game is over',
```

Comment:
As we've discussed, I think long term it would be cleaner if we separated blasting the gameover socket message from this helper and then added that logic to the consumers, but this can be a follow up.

Reply:
@Haviles04 I'd like to discuss with you to align on our approach so we can write up an issue.